packages/datadog-plugin-fastify/test/index.spec.js

```diff
@@ -0,0 +1,210 @@
+'use strict'
+
+const axios = require('axios')
+const getPort = require('get-port')
+const agent = require('../../dd-trace/test/plugins/agent')
+const plugin = require('../src')
+
+wrapIt()
+
+const sort = spans => spans.sort((a, b) => a.start.toString() >= b.start.toString() ? 1 : -1)
+
+describe('Plugin', () => {
+  let tracer
+  let fastify
+  let app
+
+  describe('fastify', () => {
+    withVersions(plugin, 'fastify', version => {
+      beforeEach(() => {
+        tracer = require('../../dd-trace')
+      })
+
+      afterEach(() => {
+        app.close()
+      })
+
+      describe('without configuration', () => {
+        before(() => {
+          return agent.load(plugin, 'fastify')
+        })
+
+        after(() => {
+          return agent.close()
+        })
+
+        beforeEach(() => {
+          fastify = require(`../../../versions/fastify@${version}`).get()
+        })
+
+        it('should do automatic instrumentation on the app routes', done => {
+          app = fastify()
```

Comment:
Can we move `app = fastify()` to a `beforeEach`?

Reply:
Done.